scripts/rollup/bundles.js

```diff
@@ -36,242 +36,88 @@ const RENDERER_UTILS = moduleTypes.RENDERER_UTILS;
 // Standalone reconciler for third-party renderers.
 const RECONCILER = moduleTypes.RECONCILER;
 
-const babelOptsReact = {
-  exclude: 'node_modules/**',
-  presets: [],
-  plugins: [],
-};
-
-const babelOptsReactART = Object.assign({}, babelOptsReact, {
-  // Include JSX
-  presets: babelOptsReact.presets.concat([
-    require.resolve('babel-preset-react'),
-  ]),
-});
-
 const bundles = [
   /******* Isomorphic *******/
   {
-    babelOpts: babelOptsReact,
+    label: 'core',
     bundleTypes: [UMD_DEV, UMD_PROD, NODE_DEV, NODE_PROD, FB_DEV, FB_PROD],
-    config: {
-      destDir: 'build/',
-      moduleName: 'React',
-      sourceMap: false,
-    },
-    entry: 'packages/react/index.js',
-    externals: [
-      'create-react-class/factory',
-      'prop-types',
-      'prop-types/checkPropTypes',
-    ],
-    fbEntry: 'packages/react/index.js',
-    hasteName: 'React',
     moduleType: ISOMORPHIC,
-    label: 'core',
-    manglePropertiesOnProd: false,
-    name: 'react',
-    paths: ['packages/react/**/*.js', 'packages/shared/**/*.js'],
+    entry: 'react',
+    global: 'React',
+    externals: [],
   },
 
   /******* React DOM *******/
   {
-    babelOpts: babelOptsReact,
+    label: 'dom-fiber',
```

Comment:
Should we drop the `-fiber` suffix from these now?

Reply:
I wanted to do in a follow up pass renaming those. Ideally I want this to be explicit and just use package names instead of labels. But that makes `yarn build react` match everything. Maybe it had to be exact match but then it's more typing. I figured let's update these later when we have a better idea of how to do this ergonomically. Not coupled to build changes per se.